cmd/soroban-cli/src/commands/env/mod.rs

```diff
@@ -7,6 +7,10 @@ use clap::Parser;
 
 #[derive(Debug, Parser)]
 pub struct Cmd {
+    /// Env variable name to get the value of.
```

Comment:
I think we should add an example, just for clarity.

```suggestion
    /// Env variable name to get the value of.
    ///
    /// E.g.: $ stellar env STELLAR_ACCOUNT
```